HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/Core/Client.swift

```diff
@@ -11,16 +11,19 @@ import Foundation
 public struct Client {
     /// Client email domain. i.e: gini.net
     public var domain: String
-    
+
+    public var apiVersion: Int
+
     /// Client id
     public var id: String
     
     /// Client secret
     public var secret: String
     
-    public init(id: String, secret: String, domain: String) {
+    public init(id: String, secret: String, domain: String, apiVersion: Int = 4) {
```

Comment:
Suggestion: Maybe add this default apiVersion as a Constant somewhere globally so that we can modify easier in the future

Reply:
good idea thanks - will implement